### PR TITLE
Updating readme to include atLeast

### PR DESCRIPTION
**Pull request by kevincol54**
Updating readme to expose GQL's atLeast function and provide end users with example.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,6 +141,19 @@ q.and([
 ]);
 ```
 
+#### At Least(number, conditions...)
```
Comment (contra) [inline README.md]: should be the function name

**Comment by contra** [comment]
Header should be a function name, otherwise looks good

**Comment by kevincol54** [comment]
Changed header to be function name

**Comment by contra** [comment]
@kevincol54 Thanks!

**Pull request merged**